core/src/style/style.cpp

```diff
@@ -95,13 +95,20 @@ void Style::setupShaderUniforms(int _textureUnit, Scene& _scene) {
 
 void Style::onBeginDrawFrame(const View& _view, Scene& _scene, int _textureUnit) {
 
+    // TODO cache which uniforms are used by the shader!
```

Comment:
Is this something we need to address? A uniform not used would at worse involve a `glUseProgram` and a hash table access. Or maybe you mean something else?